source/manage/telemetry.rst

```diff
@@ -36,32 +40,44 @@ The following data is collected once every 24 hours:
 - Date and time of the last check for security updates
 - The location of the Amazon Cloudfront server used for telemetry data
 
-To opt out, disable the feature in **System Console > Environment > SMTP**. When the feature is disabled, you will not receive any security alerts.
+Opt out
+~~~~~~~
+
+To opt out, you can disable this security update check feature for self-hosted deployments in the System Console by going to **Environment > SMTP > Enable Security Alerts**. See the `enable security alerts </configure/environment-configuration-settings.html#enable-security-alerts>`__ documentation for details. When this feature is disabled, you will not receive any security alerts.
 
 Error and diagnostics reporting feature
 ---------------------------------------
 
 Mattermost error and diagnostic data is collected for the following purposes:
+
 - To add improvements that are specific to your usage and deployment patterns, including identifying security and reliability issues.
```

Comment:
I'm unsure of what a "deployment pattern" is - I don't think we use this term regularly. Do we mean the upgrade cadence?

Reply:
Great question, @justinegeffen 

While a deployment pattern is about changes in version control triggering a process to create deployable packages and run automated unit tests and code analysis, upgrade cadence is an organization's process around how often to review systems, networks, and applications for updates that remediate security vulnerabilities.

Given that this topic is about data telemetry gathering, I feel that both concepts apply. Thoughts?

Reply:
Yeah I agree. 

"To add improvements that are specific to your usage, upgrade, and deployment patterns, including identifying security and reliability issues."

Reply:
Updated. Thank you!